crates/bitwarden-wasm-internal/src/client.rs

```diff
@@ -27,6 +27,56 @@ fn convert_level(level: LogLevel) -> Level {
     }
 }
 
+#[wasm_bindgen]
+pub struct BitwardenPure;
+
+#[wasm_bindgen]
+impl BitwardenPure {
```

Comment:
Since this is pub, could we get some docs on it so it's clear that teams should not be consuming this directly?